AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/audio/ActiveSpeakerDetectorTests.swift

```diff
@@ -138,14 +117,6 @@ class ActiveSpeakerDetectorTests: XCTestCase, ActiveSpeakerPolicy, ActiveSpeaker
         return true
     }
 
-    func subscribeToRealTimeEvents(observer: RealtimeObserver) {
```

Comment:
why is this all removed?

Reply:
I removed this because this doesn't seem necessary to have `ActiveSpeakerDetectorTest` to implement `AudioClientObserver`, and requires adding boilerplate code when we introduce new functions in `AudioClientObserver`. Let me know if I misunderstood anything.